### PR TITLE
Add other way to do Singleton in Haxe

### PR DESCRIPTION
**Pull request by T1mL3arn**

### CODE REVIEW

File: assets/content/cookbook/Design-patterns/singleton.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ class MySingleton {
...
 ```
 
+More elegant way.
+
+```haxe
+class MySingleton {
+
+  public static var instance(default, null):MySingleton = new MySingleton();
+  
+  private function new () {}  // private constructrotr
```
Comment (Gama11) [inline assets/content/cookbook/Design-patterns/singleton.md]: "constructrotr"? :)

File: assets/content/cookbook/Design-patterns/singleton.md [hunk 2]
```diff
@@ -16,6 +16,17 @@ class MySingleton {
 }
 ```
 
+More elegant way.
```
Comment (Gama11) [inline assets/content/cookbook/Design-patterns/singleton.md]: I agree.. Maybe this should just replace the other example? For @markknol to decide.

**Comment by markknol** [comment]
Yes, lets just replace the first example, there is no need to have two examples for the same pattern.

**Comment by markknol** [comment]
Thanks!!

**Pull request merged**